src/components/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,323 @@
+    font: ${theme.click.datePicker.dateOption.typography.label.disabled};
+  `}
+
+  ${({ $isSelected, theme }) =>
+    $isSelected &&
+    `
+      background: ${theme.click.datePicker.dateOption.color.background.active};
+      color: ${theme.click.datePicker.dateOption.color.label.active};
+    `}
+
+
+  text-align: center;
+
+  ${({ $isToday, theme }) =>
+    $isToday && `font: ${theme.click.datePicker.dateOption.typography.label.active};`}
+
+  &:hover {
+    ${({ $isDisabled, theme }) =>
+      `border: ${theme.click.datePicker.dateOption.stroke} solid ${
+        $isDisabled
+          ? theme.click.datePicker.dateOption.color.stroke.disabled
+          : theme.click.datePicker.dateOption.color.stroke.hover
+      }`};
+  }
+`;
+
+interface DatePickerInputProps {
+  isActive: boolean;
+  disabled: boolean;
+  id?: string;
+  selectedDate?: Date;
+}
+
+const DatePickerInput = ({
+  isActive,
+  disabled,
+  id,
+  selectedDate,
+}: DatePickerInputProps) => {
+  const defaultId = useId();
+  const formattedSelectedDate =
+    selectedDate instanceof Date ? selectedDateFormatter.format(selectedDate) : "";
+
+  return (
+    <HighlightedInputWrapper
+      $isActive={isActive}
+      disabled={disabled}
+      id={id ?? defaultId}
+    >
+      <Icon name="calendar" />
+      <InputElement
+        data-testid="datepicker-input"
+        readOnly
+        value={formattedSelectedDate}
+      />
+    </HighlightedInputWrapper>
+  );
+};
+
+interface CalendarProps {
+  closeDatepicker: () => void;
+  futureDatesDisabled: boolean;
+  selectedDate?: Date;
+  setSelectedDate: (selectedDate: Date) => void;
+}
+
+const Calendar = ({
+  closeDatepicker,
+  futureDatesDisabled,
+  selectedDate,
+  setSelectedDate,
+}: CalendarProps) => {
+  const calendarOptions: UseCalendarOptions = {
+    defaultWeekStart: 1,
+  };
+
+  if (selectedDate) {
+    calendarOptions.defaultDate = selectedDate;
+  }
+  const { body, headers, month, navigation, year } = useCalendar(calendarOptions);
+
+  const handleNextClick = (): void => {
+    navigation.toNext();
+  };
+
+  const handlePreviousClick = (): void => {
+    navigation.toPrev();
+  };
+
+  const headerDate = new Date();
+  headerDate.setMonth(month);
+  headerDate.setFullYear(year);
+
+  return (
+    <DatePickerContainer
+      data-testid="datepicker-calendar-container"
+      fillWidth={false}
+      orientation="vertical"
+      padding="sm"
+    >
+      <Container
+        justifyContent="space-between"
+        orientation="horizontal"
+      >
+        <IconButton
+          icon="chevron-left"
+          onClick={handlePreviousClick}
+          size="sm"
+          type="ghost"
+        />
+        <UnselectableTitle>{headerDateFormatter.format(headerDate)}</UnselectableTitle>
+        <IconButton
+          icon="chevron-right"
+          onClick={handleNextClick}
+          size="sm"
+          type="ghost"
+        />
+      </Container>
+      <DateTable>
+        <thead>
```

Comment:
why not using the Table component?

Reply:
On second thought, customizing it to what you want here probably going to be even more work

Reply:
Exactly right. That was my assessment as well